musicalbum.rb

```diff
@@ -0,0 +1,18 @@
+require './item'
+
+class Musicalbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify)
+    super(publish_date)
+    @on_spotify = on_spotify
+  end
+
+  private
+
+  def can_be_archived?
+    return true if supper || cover_state == 'bad'
+
```

Comment:
Correct me if I'm wrong, but is it the spelling error on line 14? Is it supposed to be super?